### PR TITLE
Disable reset and forgot power test to avoid the test failing

### PR DESCRIPTION
**Pull request by enyeal**
Move show team test together with its s component and fix errors due to its parameter. Move task input test to the folder of its component Change LabelView test name and fix errors due to change on its input parameters Fix incorrect properties on task view test

### CODE REVIEW

**Review by chartgerink — CHANGES_REQUESTED** [review]
Could you please not comment out tests temporarily and commit them to the pull request, without explanation? This affects both `src/auth/mutations/forgotPassword.test.ts` and `src/auth/mutations/resetPassword.test.ts`

Could you please proofread your posts? It is hard to follow the original comment in the PR without punctuation. It makes it harder to colloborate if we do not understand each other, and paying some more attention to the writing is something that would help greatly 😊

**Comment by enyeal** [comment]
Thanks for the suggestions. Sorry about the commit messages,  I will proofread them in the future.

I decided to comment out these two tests because they have been failing for a long time and we have not fixed them. Also, those tests are for mutations which we are not testing right now. In addition, it is a little difficult to debug the other tests due to the output screen being filled with the error messages of those two tests. So, in my opinion, those tests are not doing anything useful (as they are) and it is probably better to open a different issue to fix them.

**Comment by chartgerink** [comment]
Thanks @enyeal – and I agree, those tests need to be handled. To do it here, may not be the best option. You could raise it with @doomlab and @marton-balazs-kovacs and see what the path forward is first?

In the meantime, I run `npx vitest <filename>` to circumvent this issue while reviewing your PRs – maybe that is of help to you 😊

**Review by doomlab — COMMENTED** [review]
Not sure where this one is at? Have you fixed what Chris suggested?

**Comment by enyeal** [comment]
> Not sure where this one is at? Have you fixed what Chris suggested?

Yes, the changes that Chris suggested are already done.
